examples/c_code/main.c

```diff
@@ -0,0 +1,50 @@
+#include "individual.h"
+#include <math.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+
+double target(double x_0, double x_1) {
+    double target;
+    target = x_0 * x_1 + 1.0;
+    return target;
```

Comment:
```suggestion
    return x_0 * x_1 + 1.0;
```
i guess the compiler does this anyway, but it's a bit faster to read as well ;)